workers/exclusive-content/package.json

```diff
@@ -6,19 +6,17 @@
   "main": "dist/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "format": "prettier --write '**/*.{js,css,json,md}'",
-    "prepublish": "npm build",
-    "build": "mkdir -p ./dist && rollup -c > ./dist/index.js"
+    "format": "prettier --write '**/*.{js,css,json,md,ts}'",
+    "prepublish": "yarn build"
```

Comment:
Do we not still need `build`?
```
$ yarn install
yarn install v1.22.19
warning ../../../package.json: No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
$ yarn build
yarn run v1.22.19
warning ../../../package.json: No license field
error Command "build" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
I'm actually wondering whether we still need the prepublish since wrangler can now deal with typescript natively.